### PR TITLE
FIX: Risk from high to low message shows 14 days instead of 10 (EXPOSUREAPP-12362)

### PR DESCRIPTION
**Pull request by jurajkusnier**
### Testing
- change risk level from high to low to see a message in the home fragment
- number of days in the message should be loaded from `appConfigProvider`

### Jira Link
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12362

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt [hunk 1]
```diff
@@ -134,6 +134,22 @@ class HomeFragmentViewModel @AssistedInject constructor(
...
+            .isUserToBeNotifiedOfLoweredRiskLevel
+            .flow
+            .filter { it }
+            .onEach {
+                if (!isLoweredRiskLevelDialogBeingShown) {
+                    isLoweredRiskLevelDialogBeingShown = true
+                    events.postValue(
+                        HomeFragmentEvents.ShowLoweredRiskLevelDialogEvent(
+                            maxEncounterAgeInDays = appConfigProvider.getAppConfig().maxEncounterAgeInDays
+                        )
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt:148]: using current would be sufficient (getAppConfig may trigger a download)

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt [hunk 2]
```diff
@@ -134,6 +134,22 @@ class HomeFragmentViewModel @AssistedInject constructor(
                 )
             }
             .launchInViewModel()
+
+        tracingSettings
```
[the remark this answers is not in the document]
Comment (jurajkusnier) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt:139]: I usually try to do as few changes as possible during bugfixes :) but I change it here, it looks like it should still work as expected

**Review by chiljamgossow — APPROVED** [review]
LGTM

**Review by CV113 — APPROVED** [review]
UA approved

**Review by BMItr — APPROVED** [review]
LGTM

**Pull request merged**